### DISCUSSION TITLE
Right place to ask questions? Any further contact information?

### DISCUSSION THREAD

**Discussion by hrxn** [lzybkr/PSReadLine#3405] [Q&A] [answered]
I could not find any contact information on this repo here, like an IRC channel, a web forum or something.
Is there are proper channel for contact, if it's just about shooting some quick question?

Only this discussion tab here on GitHub, don't know if this is really in use, not sure, at least many questions seem to be unanswered.
I've used the search feature for discussions, but there are barely any meaningful results. Basically only this one here #2381

Is this the right venue to ask questions about PSReadLine, or to have discussions about it?

**Comment by daxian-dbw** [comment, marked as answer]
The `Discussions` is the place to ask quick questions, but it's supposed to be supported by the community and there is no guarantee that maintainers will respond in a timely manner. But you asked a good question, and we will try to cover this in our triage process as well.
